src/unity/toolkits/object_detection/object_detector.cpp

```diff
@@ -469,10 +487,20 @@ variant_map_type object_detector::evaluate(
     return std::accumulate(v.begin(), v.end(), 0.f) / v.size();
```

Comment:
We can remove the TODO above here, too ("TODO: Respect the `metric` argument...")